src/app/(authenticated)/user/dashboard/page.jsx

```diff
@@ -1,12 +1,237 @@
+            }
+          ]
+        }
+      }
+    );
+
+    
+  })
+
+  useEffect(() => {
+    const data = [
+      { year: 2010, count: 10 },
+      { year: 2011, count: 20 },
+      { year: 2012, count: 15 },
+      { year: 2013, count: 25 },
+      { year: 2014, count: 22 },
+      { year: 2015, count: 30 },
+      { year: 2016, count: 28 },
+    ];
+
+    new Chart(
+      document.getElementById('4'),
+      {
+        type: 'bar',
+        data: {
+          labels: data.map(row => row.year),
+          datasets: [
+            {
+              label: 'Acquisitions by year',
+              data: data.map(row => row.count)
+            }
+          ]
+        }
+      }
+    );
+
+    
+  })
+
+  useEffect(() => {
+    const data = [
+      { year: 2010, count: 10 },
+      { year: 2011, count: 20 },
+      { year: 2012, count: 15 },
+      { year: 2013, count: 25 },
+      { year: 2014, count: 22 },
+      { year: 2015, count: 30 },
+      { year: 2016, count: 28 },
+    ];
+
+    new Chart(
+      document.getElementById('5'),
+      {
+        type: 'bar',
+        data: {
+          labels: data.map(row => row.year),
+          datasets: [
+            {
+              label: 'Acquisitions by year',
+              data: data.map(row => row.count)
+            }
+          ]
+        }
+      }
+    );
+
+    
+  })
+
+  useEffect(() => {
+    const data = [
+      { year: 2010, count: 10 },
+      { year: 2011, count: 20 },
+      { year: 2012, count: 15 },
+      { year: 2013, count: 25 },
+      { year: 2014, count: 22 },
+      { year: 2015, count: 30 },
+      { year: 2016, count: 28 },
+    ];
+
+    new Chart(
+      document.getElementById('6'),
+      {
+        type: 'bar',
+        data: {
+          labels: data.map(row => row.year),
+          datasets: [
+            {
+              label: 'Acquisitions by year',
+              data: data.map(row => row.count)
+            }
+          ]
+        }
+      }
+    );
+
+    
+  })
+
+  return (
+    <>
+      <Header title="Dashboard" avatar="https://avatars.githubusercontent.com/u/102611166?s=70&v=4" />
+      <div className="h-full">
+
+        <div className="m-8">
+
+          <div className="flex-start flex-col">
+            <h2 className="mb-5 font-normal text-xl">Dashboards</h2>
+            <h3 className="font-bold text-5xl mb-3">Overview</h3>
+          </div>
+
+          <div className="grid grid-cols-4 gap-4">
+            <Overview title="Saldo Atual" value="0,00" path="#" />
+            <Overview title="Receitas" value="0,00" path="/tansacoes/receitas" />
+            <Overview title="Despesas" value="0,00" path="/tansacoes/receitas" />
+            <Overview title="Cartão de Crédito" value="0,00" path="#" />
+          </div>
+
+          <h3 className="text-3xl my-6 font-bold">Resultados</h3>
+
+          <div className="grid grid-cols-3 gap-5">
+            <div className="w-full rounded-md bg-white p-5"><canvas id="1"></canvas></div>
```

Comment:
Just a reminder: Avoid using shortened or random ID's, try to be as specific as possible.